### PR TITLE
Fixes some docs items.

### PR DESCRIPTION
**Pull request by michaelgoin**
## Proposed Release Notes

* Fixed a few small documentation items.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -140,7 +140,7 @@ If you believe you have found a security vulnerability in this project or any of
 
 If you would like to contribute to this project, review [these guidelines](./CONTRIBUTING.md).
 
-To all contributors, we thank you!  Without your contribution, this project would not be what it is today.  We also host a community project page dedicated to [Project Name](<LINK TO https://opensource.newrelic.com/projects/... PAGE>).
+To all contributors, we thank you!  Without your contribution, this project would not be what it is today.  We also host a community project page dedicated to [New Relic Next.js instrumentation](https://opensource.newrelic.com/projects/newrelic/newrelic-node-nextjs).
```
Comment (michaelgoin) [inline README.md:143]: I could not find this in the open source website, so unsure if that has merged yet. So can't confirm this link is correct. If it has been merged, then need to find the real link but I'd expect this to be the link.

Comment (bizob2828) [reply README.md:143]: it appears this hasn't been released yet. looking at their PRs they don't ship every PR but cut releases so it'll go in next one

File: docs/segments-and-spans.md [hunk 2]
```diff
@@ -23,8 +23,8 @@ If you have middleware in a deeply nested application, segments and spans will b
 If a request is made to `pages/nested/last/1`, there will be 4 segments:
```
Comment (bizob2828) [inline docs/segments-and-spans.md:23]: this URL is actually wrong should be `/nested/last/1`

**Review by bizob2828 — APPROVED** [review]

File: docs/segments-and-spans.md (hunk 2 above)
Comment (bizob2828) [reply docs/segments-and-spans.md:23]: i'll fix this in separate PR

**Pull request merged**